### ISSUE TITLE
Is it possible to setup ctrlp-funky using fzf?

### ISSUE THREAD

**Issue by rahmatawaludin** [tacahiroy/ctrlp-funky#70]
I love your plugin. But, I've found fzf is faster than ctrlp. Is it possible to implement this plugin using fzf https://github.com/junegunn/fzf ?

**Comment by tacahiroy** [comment]
Yeah - fzf is written in Go, so it's faster than ctrlp.vim written in Vim script :)
I've quick looked at fzf and I guess it's possible to use ctrlp-funky alongside fzf.
However, some works needed for it.
I'll do more investigation if I have time.

**Comment by tacahiroy** [comment]
ah - if ctrlp-funky worked alongside fzf, fzf is used just to narrow down the list.
In this case, no big effects in terms of performance, because, ctrlp-funky is slow IMHO.
So I recommend [ctrlp-cmatcher](https://github.com/JazzCore/ctrlp-cmatcher) to you atm.

**Comment by milushov** [comment]
Owh I'm wondering the same 🙈

**Comment by tracyone** [comment]
https://github.com/tracyone/fzf-funky

porting from ctrlp-funky, a little tweek, have a try!

![Image](https://github.com/user-attachments/assets/f6e393ae-3416-46eb-a6ed-b9d78af755e7)

**Comment by tracyone** [comment]
Indeed, no big effects in terms of performance, But it is convenient for someone who don't use ctrlp.vim

**Comment by tacahiroy** [comment]
Cool. The popup window support must be a good function. Actually, I'm a bit
surprised someone still uses this type of plugin other than myself. I
thought everyone is switching to the LSP's WorkspaceSymbols type of thing.

2025年2月24日(月) 14:52 tracyone ***@***.***>:

**Comment by tracyone** [comment]
The function of floating window & popup window(in vim) is provided by fzf.vim, only thing you need to do is set g:fzf_layout properly.

And the preview function is probided by fzf.vim too  (:help fzf#vim#with_preview)

It is very easy make a fzf vim extension if you can get  the list of source(s:candidates in fzf-funky). So I say it is a very little tweak.
